src/Arcus.Security.Core/SecretNotFoundException.cs

```diff
@@ -40,34 +38,9 @@ public SecretNotFoundException(string name, Exception innerException) : base($"T
             Name = name;
         }
 
-        /// <summary>
-        /// Creates <see cref="SecretNotFoundException"/> used for serialization.
-        /// </summary>
-        /// <param name="info">The <see cref="T:SerializationInfo"></see> that holds the serialized object data about the exception being thrown.</param>
-        /// <param name="context">The <see cref="T:StreamingContext"></see> that contains contextual information about the source or destination.</param>
-        /// <exception cref="ArgumentNullException">The info must not be <c>null</c>.</exception>
-        /// <exception cref="SerializationException">The class name must not be <c>null</c> and <see cref="Exception.HResult"/> must not be zero (<c>0</c>).</exception>
-        protected SecretNotFoundException(SerializationInfo info, StreamingContext context) : base(info, context)
-        {
-            Name = info.GetString(nameof(Name));
-        }
-
         /// <summary>
         /// Name of the missing key
         /// </summary>
         public string Name { get; } = "undefined";
-
-        /// <summary>
-        /// When overridden in a derived class, sets the <see cref="T:SerializationInfo"></see> with information about the exception.
-        /// </summary>
-        /// <param name="info">The <see cref="T:SerializationInfo"></see> that holds the serialized object data about the exception being thrown.</param>
-        /// <param name="context">The <see cref="T:StreamingContext"></see> that contains contextual information about the source or destination.</param>
-        /// <exception cref="T:ArgumentNullException">The <paramref name="info">info</paramref> parameter is a null reference (Nothing in Visual Basic).</exception>
-        public override void GetObjectData(SerializationInfo info, StreamingContext context)
-        {
-            base.GetObjectData(info, context);
-
-            info.AddValue(nameof(Name), Name);
-        }
```

Comment:
This is no longer necessary with .NET 8 ?

Reply:
Yes, as far as I can see: https://github.com/dotnet/docs/issues/34893

Reply:
It gave an obsolete message, the `GetObjectData(...)` call.